desktop/cmp/grid/columns/Actions.js

```diff
@@ -70,7 +69,7 @@ export const actionCol = {
             if (!isEmpty(tooltip)) buttonEl.setAttribute('title', tooltip);
             if (!isEmpty(intent)) buttonEl.classList.add(`bp3-intent-${intent}`);
 
-            buttonEl.innerHTML = convertIconToSvg(icon);
```

Comment:
again problematic here -- my guess is that you need to revert this code to get the html you need here.

Reply:
This also works if RecordAction takes an icon that has config {asSvg: true}. Not sure how to document that change more clearly (and also why this is a breaking change).
See RecordAction.js line 46: ` // @param {Element} [c.icon] - icon to be displayed. Always specify with config {asSvg: true}.`

Reply:
Ah -- I see, but then we are lying with jsdoc {Element} declaration on line 46 -- its not an element, its a string of html.   This is a tricky bit, lets dicsuss tommorrow.